2025R2/dpf-operator-spec-25-r2/core-concepts/dpf-types.md

```diff
@@ -132,48 +132,133 @@ For cyclic simulation models, the ``cyclic_support`` holds information about the
 
 ### Filtering
 
+<a id="filtering"></a>
+
+When handling large amounts of simulation data, you can select a subset of the data and filter it down to the data of interest to the analysis.
+
+In the DPF ecosystem this is called ``scoping`` the data, and is managed with ``scoping`` entities.
+
+#### Scoping
+
 <a id="scoping"></a>
 
-- scoping
+A ``scoping`` describes a typed list of IDs. 
+
+It allows you to select and filter data when given as input to operators, or data description when produced as output.
+
+Each ``field`` data storage type has a ``scoping`` associated to it, describing the subset of its ``support`` that the data applies to. The data in a ``field`` is ordered the same way as the IDs in the ``scoping``, allowing you to reorder the data.
+
+It has a ``location``, which defines the type of entity the IDs apply to (nodes, elements, time-steps, parts...), and a list of ``ids``.
 
 ### Data storage
 
+<a id="data storage"></a>
+
+The following DPF types allow you to store and describe data.
+
```

Comment:
We make reference to meshes below. We should have a section for it.

Reply:
@rafacanton the `meshed_region` is already described above. I do not understand.

Reply:
Yup, apologies